src/tests/multihost/alltests/conftest.py

```diff
@@ -1057,6 +1057,59 @@ def remove_journalsssd():
     request.addfinalizer(remove_journalsssd)
 
 
+@pytest.fixture(scope='class')
+def sssdproxyldap_test(session_multihost, request):
+    """ Configure  sssdproxyldap
+        Configure  sssd.conf
+        Configure  nslcd.conf
+        Transport sssdproxyldap.sh to client machine
+        configure password for ldap user
+    """
+    tools = sssdTools(session_multihost.client[0])
+    execute_cmd(multihost, "yum install -y nss-pam-ldapd")
+    execute_cmd(session_multihost, "> /etc/pam.d/sssdproxyldap")
+    execute_cmd(session_multihost, "echo 'auth        required      "
+                                   "pam_ldap.so' > /etc/pam.d/sssdproxyldap")
+    execute_cmd(session_multihost, "echo 'account     required      "
+                                   "pam_ldap.so' >> /etc/pam.d/sssdproxyldap")
+    execute_cmd(session_multihost, "echo 'password    required      "
+                                   "pam_ldap.so' >> /etc/pam.d/sssdproxyldap")
+    execute_cmd(session_multihost, "echo 'session     required      "
+                                   "pam_ldap.so' >> /etc/pam.d/sssdproxyldap")
```

Comment:
There is `sssdproxyldap` fixture that does the same as  above lines using a static file, it will be better to modify and use that

Reply:
done